package.json

```diff
@@ -13,7 +13,8 @@
     "jsonwebtoken": "^9.0.2",
     "next": "14.1.0",
     "react": "^18",
-    "react-dom": "^18"
+    "react-dom": "^18",
+    "react-hook-form": "^7.52.1"
```

Comment:
this is not in use, right?

Reply:
Nice, but after you remove the package you should also update/restore the lockfile - could you do that?